### PR TITLE
Fix a path issue in the README.md

### PR DESCRIPTION
**Pull request by irbull**
The instructions say 'go into the repository root folder' (which is ebr/), 
but then ebr/ is used in the path on the mvn build line.

### CODE REVIEW

**Comment by guw** [comment]
@irbull did you see the fancy "Merge with caution" warning? :)

**Comment by irbull** [comment]
Oh right... Eclipse projects (this is my first pull request for an Eclipse project). Let me fix that. :)

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ any other accessible Maven repository. A Maven plug-in is provided which must be
 in your local Maven repository for any of the builds to be successful.
 
 1. Install Java (at least Java 7) and Maven
-2. Clone this repository and go into the repository root folder.
-3. `mvn -f ebr/releng/ebr-maven-plugin/pom.xml install`
+2. Clone this repository and go into the repository root folder
+3. `mvn -f releng/ebr-maven-plugin/pom.xml install`
```
Comment (guw) [inline README.md]: I wonder if that is still necessary, now that we publish the plug-in to repo.eclipse.org.

File: recipes/square/com.squareup.okhttp_2.3.0/.project [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.google.guava_16.0.1</name>
```
Comment (irbull) [inline recipes/square/com.squareup.okhttp_2.3.0/.project]: I copied the guava one as a template. I can fix that.

Comment (guw) [inline recipes/square/com.squareup.okhttp_2.3.0/.project]: Thanks! Please do. You should really try the Maven command next time. It auto-generate everything including about files and license info.

**Pull request merged**